app/api/events.py

```diff
@@ -525,6 +531,7 @@ def before_update_object(self, event, data, view_kwargs):
         :param view_kwargs:
         :return:
         """
+        EventDetail.event_name = event.name
```

Comment:
You don't need to do this, and this is wrong, You have data and obj in after update object, use that

Reply:
How will i get the previous event name for comparison? Also, what is obj in after update object?

Reply:
Use before update object and flask.g